src/Notification/Notification.js

```diff
@@ -1,8 +1,16 @@
 import React, {Component} from 'react';
 class Notification extends Component {
+
+    componentWillReceiveProps(nextProps) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "nextProps". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)